docs/source/examples/simple-landscape-braket.md

```diff
@@ -0,0 +1,203 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Using ZNE to compute the energy landscape of a variational circuit with Braket
+
+This tutorial shows an example in which the energy landscape for a two-qubit variational circuit is explored with and without error mitigation, using Braket as our frontend.
```

Comment:
We should add a link to braket documentation for those who are not super familiar.
```suggestion
This tutorial shows an example in which the energy landscape for a two-qubit variational circuit is explored with and without error mitigation, using Amazon's [Braket](https://amazon-braket-sdk-python.readthedocs.io/en/latest/) as our frontend.
```
Do you think this is the correct link? There are a few I could imagine using.

Reply:
Yeah this link is fine. Perhaps this [link](https://docs.aws.amazon.com/braket/latest/developerguide/braket-using.html) also works? I consulted this one when writing the notebook but the readthedocs page is probably more frequently updated.

Reply:
You've looked into this more than I have, so happy for you to choose what you think is best!